packages/console/src/components/Authenticator.tsx

```diff
@@ -58,6 +58,8 @@ export function AuthenticationSubmitted (): JSX.Element {
         <h1 className='text-xl font-epilogue'>Verify your email address!</h1>
         <p className='pt-2 pb-4'>
           Click the link in the email we sent to <span className='font-semibold tracking-wide'>{email}</span> to authorize this agent.
+          <br />
+          Don't forget to check your spam folder!
```

Comment:
Linter's mad, gotta escape those entities!

```suggestion
          Don&apos;t forget to check your spam folder!
```